### PR TITLE
Enable new page feedback template in hugo

### PR DESCRIPTION
**Pull request by svrnm**
follow up to https://github.com/open-telemetry/opentelemetry.io/pull/5548

### CODE REVIEW

File: hugo.yaml [hunk 1]
```diff
@@ -114,8 +114,8 @@ params:
         Thank you. Your feedback is appreciated!
       # prettier-ignore
       'no': >-
-        Please let us know <a class="external-link" target="_blank" rel="noopener"
-        href="https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Page%20feedback&body=Suggested%20improvements%20for%20page:%20ADD%20PAGE-URL-HERE">how
+        Please let us know <a class="external-link feedback-link" target="_blank" rel="noopener"
```
Comment (chalin) [inline hugo.yaml:117]: What is the `feedback-link` class for?

Comment (svrnm) [reply hugo.yaml:117]: Ah, I was shortly thinking about doing some javascript hack to update the url parameter with the current url but then decided against it and waiting for docsy to provide that as a hugo variable as an upstream feature ;-)

File: hugo.yaml [hunk 2]
```diff
@@ -115,7 +115,7 @@ params:
       # prettier-ignore
       'no': >-
         Please let us know <a class="external-link" target="_blank" rel="noopener"
-        href="https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Page%20feedback&body=Suggested%20improvements%20for%20page:%20ADD%20PAGE-URL-HERE">how
+        href="https://github.com/open-telemetry/opentelemetry.io/issues/new?template=PAGE_FEEDBACK.yml&title=[Page+Feedback]%3A+summary+of+your+feedback">how
```
Comment (chalin) [inline hugo.yaml:118]:
```suggestion
        href="https://github.com/open-telemetry/opentelemetry.io/issues/new?template=PAGE_FEEDBACK.yml&title=[Page+feedback]%3A+ADD+A+SUMMARY+OF+YOUR+FEEDBACK+HERE">how
```

**Review by chalin — APPROVED** [review]
LGTM. See inline for suggested tweak.

**Pull request merged**